browser/about_flags.cc

```diff
@@ -506,6 +507,19 @@
                    FEATURE_VALUE_TYPE(kExtensionsManifestV2),               \
                }))
 
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
why is this behind a !IS_ANDROID flag?

Reply:
It's not meant to be available on Android. I'm not sure that it can be built on Android in any case since `browser_command_handler.cc` is [not built on Android](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/BUILD.gn;l=1652?q=%22browser_command_handler.h%22&ss=chromium).

Reply:
That's not really an appropriate use of an os buildflag. If the entire feature is not available on android then it should have its own buildflag with `= !is_android`. OS buildflags should be reserved for os-specific code and also the feature flag itself is not behind any buildflag so this doesn't really make sense here as-is.